postgresql/provider.go

```diff
@@ -43,7 +44,7 @@ func Provider() *schema.Provider {
 				Type:        schema.TypeString,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("PGHOST", nil),
-				Description: "Name of PostgreSQL server address to connect to",
+				Description: "Name of PostgreSQL server address",
```

Comment:
Would revert this change, I feel "to connect to" is useful in distinguishing the difference between the options

Reply:
done